### DISCUSSION TITLE
Cant load any jails but sshd, need postfix, postfix-sasl

### DISCUSSION THREAD

**Discussion by wspivak** [fail2ban/fail2ban#3409] [Q&A] [answered]
Just updated to Fedora 37.  Under 34,35 and 36 tried each time to implement fail2ban and it failed.  I really don't need sshd since I use a non-standard port.

I've been getting hit with large number of SASL Auth errors.

I noticed that postfix-sasl was renamed, but a) I don't have that filter and b) it's not in the master Git file.

Assistance appreciated.

**Comment by wspivak** [comment]
Found the partial solution.  Re-copied jail.conf to jail.local and re-edited.

Now waiting to see if it catches any miscreants.

**Reply by sebres** [reply]
> Re-copied jail.conf to jail.local and re-edited.

FWIW, this is not so good approach - `jail.local` basically expects only parameters that you want to overwrite. Copying of `jail.conf` to `jail.local` completely may introduce some issues by later upgrades (if `jail.conf` becomes changed, e. g. some default values that you'd normally never touch).

**Comment by wspivak** [comment]
Here is a SASL failure.  It wasn't caught by Fail2ban

Nov 20 08:08:02 mcq postfix/smtpd[13392]: warning: unknown[12.188.54.30]: SASL LOGIN authentication failed: UGFzc3dvcmQ6

This is the standard in postfix.conf:
mdpr-auth = warning:
mdre-auth = ^[^[]*\[<HOST>\]%(_port)s: SASL ((?i)LOGIN|PLAIN|(?:CRAM|DIGEST)-MD5) authentication failed:(?! Connection lost to authentication server| Invalid authentication mechanism)
mdre-auth2= ^[^[]*\[<HOST>\]%(_port)s: SASL ((?i)LOGIN|PLAIN|(?:CRAM|DIGEST)-MD5) authentication failed:(?! Connection lost to authentication server)
# todo: check/remove "Invalid authentication mechanism" from ignore list, if gh-1243 will get finished (see gh-1297).

What do I need to change to make it work.

Help appreciated.

Thanks,
Wayne

**Comment by wspivak** [comment, marked as answer]
I solved my issue:

I followed this: https://serverfault.com/questions/1099716/postfix-couldnt-stop-sasl-login-authentication-failed-attack-in-mail-logs

I ran  fail2ban-regex -v sasl.lines /etc/fail2ban/filter.d/postfix-sasl --print-all-missed and final results were:

Lines: 34 lines, 0 ignored, 34 matched, 0 missed

**Comment by wspivak** [comment]
Thank you.

 

Finally got it to work properly.

 

Happy New Year!
